.github/workflows/build-toolchain.yml

```diff
@@ -933,6 +935,7 @@ jobs:
       ANDROID_NDK_VERSION: ${{ needs.context.outputs.ANDROID_NDK_VERSION }}
       CMAKE_Swift_FLAGS: ${{ needs.context.outputs.DARWIN_CMAKE_Swift_FLAGS }}
       debug_info: ${{ fromJSON(needs.context.outputs.debug_info) }}
+      release: false
```

Comment:
Why is this unconditionally false?

Reply:
This is for the Mac build, which is still very experimental. There is almost no risk to setting this the same way we do for Windows since the relevant jobs are disabled for macOS but I would rather err on the side of caution.